plugin/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id("org.jetbrains.kotlin.jvm") version "1.7.21"
+    `java-gradle-plugin`
+    `maven-publish`
+}
+
+group = "tmg.gradle"
```

Comment:
Should we make this a release version?

Reply:
I was going to start with snapshots, get it publishing to github packages, test it in the app, and then tag it as a release

Reply:
this will result in a published artifact of `tmg.gradle:plugin:1.0-SNAPSHOT` which might be a little weird...

But to be fair, what really matters is the plugin id (line 13 👇) so as long as we're publishing that, people won't even have to know about the group:artifact